packages/fxa-auth-server/test/local/payments/stripe.js

```diff
@@ -4264,6 +4263,77 @@ describe('StripeHelper', () => {
         'extracts expected details for a subscription downgrade',
         commonTest(false)
       );
+
+      it('checks productPaymentCycleOld returns a value if it is not included in the old plan', async () => {
+        const event = deepCopy(eventCustomerSubscriptionUpdated);
+
+        // if the interval of old and new plans are the same,
+        // the old plan's previous_attributes object may not include interval value.
+        // this update `event` to remove its interval value
```

Comment:
nit: maybe a typo? should it read `use this update...` as opposed `this update`.